app/views/articles/_articles.html.slim

```diff
@@ -0,0 +1,27 @@
+table
+  tr
+    th Title
+    th Text
+    th colspan="3"
+  - @articles.each do |a|
```

Comment:
Remember not to use variables with @ in partials. There is a way to send an object from the controller